File: SGA_Api/SGA_Api/Models/Impresion/LogImpresionDto.cs
namespace SGA_Api.Models.Impresion
{
	public class LogImpresionDto
	{
		public string Usuario { get; set; }
		public string Dispositivo { get; set; }
		public int IdImpresora { get; set; }
		public int EtiquetaImpresa { get; set; }
		public int? Copias { get; set; } // Opcional, backend controla que por defecto sea 1
		public string? CodigoArticulo { get; set; }
		public string? DescripcionArticulo { get; set; }
		public string? CodigoAlternativo { get; set; }
		public DateTime? FechaCaducidad { get; set; }
		public string? Partida { get; set; }
		public string? Alergenos { get; set; }
		public string PathEtiqueta { get; set; }

		public int TipoEtiqueta { get; set; }
		public string? CodigoGS1 { get; set; }
		public string? CodigoPalet { get; set; }

		// Campos de ubicación
		public string? CodAlmacen { get; set; }
		public string? CodUbicacion { get; set; }
		public int? Altura { get; set; }
		public int? Estanteria { get; set; }
		public int? Pasillo { get; set; }
		public int? Posicion { get; set; }
	}
}

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/etiquetas/LogImpresionDto.kt
package com.example.sga.data.dto.etiquetas

import com.google.gson.annotations.SerializedName
import java.time.LocalDate

data class LogImpresionDto(
    val usuario: String,
    val dispositivo: String,
    val idImpresora: Int,
    val etiquetaImpresa: Int,
    val codigoArticulo: String? = null,
    val descripcionArticulo: String? = null,
    val copias: Int = 1,
    val codigoAlternativo: String? = null,
    val fechaCaducidad: String? = null,
    val partida: String? = null,
    val alergenos: String? = null,
    val pathEtiqueta: String? = null,
    @SerializedName("TipoEtiqueta")
    val tipoEtiqueta: Int? = null,
    @SerializedName("CodigoGS1")
    val codigoGS1: String? = null,
    @SerializedName("CodigoPalet")
    val codigoPalet: String? = null
)